releaseNotes/AzureDevOpsPushPipelineExtension.md

```diff
@@ -0,0 +1,9 @@
+## Azure App Configuration Push Extension release notes
+The Azure App Configuration Push extension in Azure DevOps pipeline has been published to

```

Comment:
How about "~~The~~ Azure App Configuration Push extension *for* Azure DevOps pipeline *can be installed from*..."?

Reply:
Any reason we are still saying "published"?